p2p/libp2p/netMessenger_test.go

```diff
@@ -1799,3 +1799,50 @@ func TestNetworkMessenger_Bootstrap(t *testing.T) {
 	goRoutinesNumberStart := runtime.NumGoroutine()
 	core.DumpGoRoutinesToLog(goRoutinesNumberStart, log)
 }
+
+func TestNetworkMessenger_WaitForConnections(t *testing.T) {
```

Comment:
Shall we add another test in which no peer will connect within the waiting time-frame? Or are there other tests in this file that take care of it?

Reply:
added test